src/effects/specialeffectchainslots.h

```diff
@@ -0,0 +1,70 @@
+#ifndef SPECIALEFFECTCHAINSLOTS_H
+#define SPECIALEFFECTCHAINSLOTS_H
```

Comment:
pragma once

Reply:
fixed in 9d84e26a2a0118c03e778a35015f471cee588092